google/resource_google_project_test.go

```diff
@@ -167,9 +111,8 @@ func TestAccGoogleProject_updateBilling(t *testing.T) {
 	})
 }
 
-// Test that a Project resource merges the IAM policies that already
-// exist, and won't lock people out.
-func TestAccGoogleProject_merge(t *testing.T) {
```

Comment:
This is already tested in TestAccGoogleProjectIamPolicy_basic in the resource_google_project_iam_policy_test.go file